one-ui/src/test/java/com/marklogic/hub/curation/controllers/EnvironmentControllerTest.java

```diff
@@ -248,6 +253,29 @@ public void doAppend(ILoggingEvent event) throws LogbackException {
         assertTrue(SUCCESS_LOG_MSG.isEmpty(), "has error or exception thrown.");
     }
 
+    @Test
+    public void testManageAdminAndSecurityAuthoritiesForArtifacts() {
+        testHelper.authenticateSessionAsEnvironmentManager();
+        ArtifactManager mgr = testHelper.getArtifactManager();
+        List<ArtifactTypeInfo> listTypeInfo = mgr.getArtifactTypeInfoList();
+        for (ArtifactTypeInfo typeInfo : listTypeInfo) {
+            assertTrue(typeInfo.getUserCanUpdate());
+            assertTrue(typeInfo.getUserCanRead());
+        }
+    }
+
+    @Test
+    public void testAdminAuthoritiesForArtifacts()  {
+        EnvironmentInfo environmentInfo = new EnvironmentInfo("localhost", "DIGEST", 8000,"DIGEST", 8002,"DIGEST", 8010, "DIGEST", 8011);
+        hubConfigSession.setCredentials(environmentInfo, testHelper.adminUserName, testHelper.adminPassword);
+        ArtifactManager mgr = testHelper.getArtifactManager();
+        List<ArtifactTypeInfo> listTypeInfo = mgr.getArtifactTypeInfoList();
+        for (ArtifactTypeInfo typeInfo : listTypeInfo) {
+            assertTrue(typeInfo.getUserCanUpdate());
+            assertFalse(typeInfo.getUserCanRead());
```

Comment:
Can you add an assertion message to explain why an admin user can update, but not read? That of course is surprising behavior - normally a user can read but not update (or can do both). According to the JIRA ticket, it's to allow an admin to write artifacts during deployment - I think it's important that an assertion message captures that.